ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -703,6 +691,22 @@ TFLiteInterpreter::cacheInOutTensorPtr ()
   return -EINVAL;
 }
 
+/**
+ * @brief	TFLiteCore constructor
+ */
+TFLiteCore::TFLiteCore ()
+{
+  interpreter = new TFLiteInterpreter ();
```

Comment:
```TFLiteInterpreter``` looks like a struct without any constructors (default one would be invoked at this line).

Then, are there any reasons why we do not use ```TfLiteInterpreterCreate```?

https://github.com/tensorflow/tensorflow/blob/7ce597b5e8467cb662c9ae9a968e1ff10741f734/tensorflow/lite/c/c_api.cc#L131

Reply:
The class `TFLiteInterpreter` is a different one you shared. It's a respective class.
PTAL https://github.com/nnstreamer/nnstreamer/blob/f1257d7c4d54699de88ce868f446a461084abb9e/ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc#L117